src/TPLinkDevice.php

```diff
@@ -62,15 +62,15 @@ public function sendCommand(array $command)
      */
     protected function connectToDevice()
     {
-        $this->client = stream_socket_client(
+        $this->client = @stream_socket_client(
```

Comment:
Why are we suppressing warnings here? I'm not keen on that. If there's an error generated we should either deal with it or tell the user.